spec/daru/io/exporters/csv_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'spec_helper'
+require 'tempfile'
+require 'open-uri'
+
+RSpec.describe Daru::IO::Exporters::CSV do
+  context "writes DataFrame to a CSV file" do
+  let(:df) { 
+    Daru::DataFrame.new({
+      'a' => [1,2,3,4,5],
+      'b' => [11,22,33,44,55],
+      'c' => ['a', 'g', 4, 5,'addadf'],
+      'd' => [nil, 23, 4,'a','ff']})
+  }
+  let(:tempfile) { Tempfile.new('data.csv') }
+
+    before { Daru::IO::Exporters::CSV.write df, tempfile.path }
+    subject { Daru::IO::Importers::CSV.load tempfile.path }
```

Comment:
Also, I disapprove on using _importer_ to test _exporter_. It is higly probably to have symmetric bugs in them, hiding each other. I advise to have just `CSV.read(...)` as a subject.